src/Mvc/Mvc.Core/src/Routing/HttpMethodAttribute.cs

```diff
@@ -63,6 +66,7 @@ public int Order
         int? IRouteTemplateProvider.Order => _order;
 
         /// <inheritdoc />
-        public string Name { get; set; }
+        [DisallowNull]
+        public string? Name { get; set; }
```

Comment:
What does combining string? + DisallowNull do?

Reply:
It says while this could return `null`, you shouldn't assign a `null` value to it. Essentially the setter is treated as non-null